backend/QA/src/main/java/com/example/qa/dao/question/QuestionDao.java

```diff
@@ -188,6 +188,13 @@ public void update(
       throws DaoException {
     Put put = createPutWithForUpdate(date, createdAt, updatedAt, numAnswer);
 
+    /** To manage partial updates on Cosmos DB, Cosmos DB does not currently support partial
+     *  document updates*/

```

Comment:
```suggestion
    /** Read the updating record and update all values for a workaround of the partial update
      * because Cosmos DB does not support partial updating.
      * For other data stores, this is not needed. */
```

Reply:
Thank you for the suggestion.
Document update accordingly.